File: URI/Python/idade-em-dias.py
dias = int(input())

ano = dias // 365
mes = (dias - (ano * 365)) // 30
dia = dias - ((ano*365) + (mes * 30))

print("""{} ano(s)
{} mes(es)
{} dia(s)""".format(ano, mes, dia))

File: URI/C/idade-em-dias.c
#include <stdio.h>
#include <stdlib.h>

int main(void){

    int dias, dia, mes, ano;
    scanf("%d", &dias);

    ano = dias / 365;
    mes = (dias - (ano * 365)) / 30;
    dia = dias - ((ano * 365) + (mes * 30));

    printf("%d ano(s)\n%d mes(es)\n%d dia(s)\n", ano, mes, dia);

    return 0;
}